clvm_tools/cmds.py

```diff
@@ -205,15 +218,34 @@ def launch_tool(args, tool_name, default_stage=0):
     try:
         output = "(didn't finish)"
 
-        use_rust = (tool_name != "run") and not args.table and not args.stage and (args.backend == "rust" or (serialize_and_run_program and args.backend != "python"))
+        use_rust = (
+            (tool_name != "run")
+            and not pre_eval_f
```

Comment:
Note that `args.stage` was always True, so removing it is quite important.

Reply:
Thanks